pyrightconfig.json

```diff
@@ -22,4 +22,5 @@
     "reportUndefinedVariable": "warning",
     "reportOptionalOperand": "warning",
     "reportMissingImports": "warning",
+    "reportSelfClsParameterName": "none",
```

Comment:
What's the problem with this check?

Reply:
Sorry, I lost the commit with the comment, I'll find it.

Reply:
The comment is back and does not need further clarification